polyglot/piranha/src/models/matches.rs

```diff
@@ -41,5 +38,40 @@ impl Match {
   pub(crate) fn matches(&self) -> &HashMap<String, String> {
     &self.matches
   }
+}
+
+fn ser_range<S: Serializer>(range: &Range, serializer: S) -> Result<S::Ok, S::Error> {
```

Comment:
Can you add some comment on what we are serializing?

Reply:
Addressed.